deployment/tfhub/object-detection-url.ipynb

```diff
@@ -0,0 +1,607 @@
+        "    \n",
+        "    def handle_img(self, file_name, url, width=640, height=480):\n",
+        "        wget.download(url, file_name)\n",
+        "        img = Image.open(file_name)\n",
+        "        img = ImageOps.grayscale(img)\n",
+        "        img.thumbnail((width, height), Image.LANCZOS)\n",
+        "\n",
+        "    def load_img(self, path):\n",
+        "        img = tf.io.read_file(path)\n",
+        "        img = tf.image.decode_jpeg(img, channels=3)\n",
+        "        \n",
+        "        return img\n",
+        "\n",
+        "    def filter_results(self, file_name, response, entity='Car', min_score=.2):\n",
+        "        unused_keys = ['detection_class_labels', 'detection_class_names']\n",
+        "        response = {key: value.numpy().tolist() for key, value in response.items()}\n",
+        "        response = {key: val for key, val in response.items() if key not in unused_keys}\n",
+        "        response['detection_class_entities'] = [v.decode() for v in response['detection_class_entities']]\n",
+        "\n",
+        "        entities = response['detection_class_entities']\n",
+        "        scores = response['detection_scores']\n",
+        "        bboxes = response['detection_boxes']\n",
+        "        result = {}\n",
+        "\n",
+        "        for i in range(len(entities)):\n",
+        "            if entities[i] == entity and scores[i] >= min_score:\n",
+        "                ymin, xmin, ymax, xmax = bboxes[i]\n",
+        "                result = {\n",
+        "                    'file_name': file_name,\n",
+        "                    'has_car': True,\n",
+        "                    'score': scores[i],\n",
+        "                    'bboxes': {'ymin': ymin, 'xmin': xmin, 'ymax': ymax, 'xmax': xmax}\n",
+        "                }\n",
+        "                break\n",
+        "\n",
+        "        if len(result) == 0:\n",
+        "            result = {\n",
+        "                'file_name': file_name,\n",
+        "                'has_car': False,\n",
+        "                'score': 0,\n",
+        "                'bboxes': {'ymin': 0, 'xmin': 0, 'ymax': 0, 'xmax': 0}\n",
+        "            }\n",
+        "            \n",
+        "        return result\n",
+        "\n",
+        "    def run_detector(self, file_name, image_path):\n",
+        "        img = self.load_img(image_path)\n",
+        "        img = tf.image.convert_image_dtype(img, tf.float32)[tf.newaxis, ...]\n",
+        "        \n",
+        "        response = self.detector(img)\n",
+        "        result = self.filter_results(file_name, response)\n",
+        "        \n",
+        "        return result\n",
+        "\n",
+        "    def detect_objects(self, file_name, url):\n",
+        "        start_time = time.time()\n",
+        "        self.handle_img(file_name, url)\n",
+        "        result = self.run_detector(file_name, file_name)\n",
```

Comment:
This looks very strange to pass the same variable for both arguments. Can we change it to just

```suggestion
        "        result = self.run_detector(file_name)\n",
```

?

Reply:
Yes, the second arg was unnecessary. Done.